cmd/cmd.go

```diff
@@ -1197,9 +1208,33 @@ func getRepo(repo string) (string, error) {
 		if err := os.MkdirAll(appDir, 0755); err != nil {
 			return "", fmt.Errorf(fmt.Sprintf("create repo directory failed: %s", err))
 		}
-		repo = filepath.Join(appDir, "repo")
+		baseRepo = filepath.Join(appDir, "repo")
+	}
+	return baseRepo, nil
+}
+
+// Get the full repo path for the user, will use the first
+// directory inside baseRepo if accountSeed isn't provided
+func getRepo(baseRepo string, accountSeed string) (string, error) {
+	if len(accountSeed) == 0 {
+		files, err := ioutil.ReadDir(baseRepo)
+		if err != nil {
+			return "", err
+		}
+		if len(files) == 0 {
+			return "", fmt.Errorf("no account repos initialized in: %s", baseRepo)
+		}
+		return path.Join(baseRepo, files[0].Name()), nil
+	}
+	kp, err := keypair.Parse(accountSeed)
+	if err != nil {
+		return "", fmt.Errorf(fmt.Sprintf("parse account seed failed: %s", err))
```

Comment:
nit, Errorf takes a format string, so the inner Sprintf is not required

Reply:
Cool, fixed it here and everywhere else in the repo

Reply:
oh nice, i must have been the origin of that ha